src/components/About.jsx

```diff
@@ -0,0 +1,12 @@
+// src/components/About.jsx
+import React from 'react';
+
+const About = () => {
+  return (
+    <div className="w-full h-screen flex items-center justify-center">
+      <h1 className="text-white text-4xl">About Us</h1>
+    </div>
```

Comment:
If this component is not serving any use at the moment, let's remove it

Reply:
As of right now this is a place holder for the info tab (just so i could see if the navbar loaded the page)